app/views/users/profile.html.erb

```diff
@@ -239,6 +244,15 @@
     alert("Your access token is: <%= @user.user.token %>");
   });
 
+$('#link_social_media').click(function(){
+   var sign = prompt("Enter Your Social Media User name like twitter:jywarren or instagram:unterbahn");
```

Comment:
Sorry to ask for just one more thing -- could we tidy up the indenting here so it's two-spaces per level, and would you mind changing the `jywarren` and `unterbahn` to `publiclab` both? Thank you! 

Also, based on this line, we'll see an alert that the tag is saved, so I don't think we need an `alert()` in addition:

https://github.com/publiclab/plots2/blob/53d83d45a19df533f7420a49671334ee6a810783/app/controllers/user_tags_controller.rb#L46

Thanks, this is looking super!